websites/D/Deezer/presence.ts

```diff
@@ -91,6 +97,20 @@ presence.on("UpdateData", async () => {
       presenceData.state = document.querySelector(
         ".track-link:nth-child(2)"
       ).textContent;
+
+      albumCoverID ??= albumID;
+      albumCoverURL ??= (
+        await fetch(`https://api.deezer.com/album/${albumCoverID}/image`)
+      ).url;
+
+      if (albumCoverID !== albumID) {
+        albumCoverID = albumID;
+        albumCoverURL = (
+          await fetch(`https://api.deezer.com/album/${albumCoverID}/image`)
+        ).url;
+      }
+
+      presenceData.largeImageKey = cover ? albumCoverURL : "deezer";
```

Comment:
`deezer` is already set on L9. Use the nullish operator instead
```suggestion
      presenceData.largeImageKey = cover ?? albumCoverURL;
```

Reply:
This won't work

Reply:
agreed